src/index.js

```diff
@@ -1,4 +1,4 @@
-import './index.css';
+import './style.css';
```

Comment:
- [ ] Good job implementing the project requirements, There is only just this part where our CSS, I assume, was moved to the `modules` directory by mistake, Let's make sure we either move it back to the `src` directory or change this import to
```js
import './modules/style.css`;
```
Because our project will not run given this state, I solved it locally to be able to test the rest of requirements